packages/amplify-codegen/src/constants.js

```diff
@@ -7,10 +7,12 @@ module.exports = {
   PROMPT_MSG_GQL_FILE_PATTERN:
     'Enter the file name pattern of graphql queries, mutations and subscriptions',
   PROMPT_MSG_GENERATE_CODE: 'Do you want to generate code for your newly created GraphQL API',
+  PROMPT_MSG_CHANGE_REGION: 'Do you want to choose a different region',
```

Comment:
According to me, this message should be like. 
Could not find API in <region>. Do you want to choose a different region?

Reply:
We do show that API was not found and then we show this question asking them if they want choose a different region. So the question looks like this
```
amplify codegen add --apiId hahq7hxq5fbsto3ldozg6ewaje
✖ Getting API details
AppSync API was not found in region us-east-1
? Do you want to choose a different region Yes
? Choose AWS Region
  US East (N. Virginia)
❯ US East (Ohio)
  US West (Oregon)
  EU (Ireland)
  EU (Frankfurt)
  Asia Pacific (Tokyo)
  Asia Pacific (Singapore)
```